### ISSUE TITLE
category of the bot

### ISSUE THREAD

**Issue by SnowJuli** [dev-schueppchen/bot#3]
a question what kind of bot it‘ll be. should it be a moderation bot or a fun bot etc.

**Comment by ghost** [comment]
In my opinion:
It should be a moderation bot which also has fun sides with a few games that you can play alone or with several people. The interplay request can be made via private news and reactions. e.g. solitaire one could possibly. or if it is possible maumau. So everything in one offered a versatile.

**Comment by Ron31** [comment]
Categorys:
Games and Fun
and any other things we like. But moderation is the exercise from Shinpuru.
Something like Hacktoberfest is possible, too. Sth. like count to 100 is possible, too. Everything we want, but not moderation. *Hopefully my English is okay in this text*

**Comment by vbe0201** [comment]
I wouldn't focus on it too hard. As an ongoing community project, the bot is mainly a reflection of our behavior from time to time. You all know how this community works, stupid, super weird or even brilliant ideas are born within seconds. And this holds true for the bot as well.

Tl;dr: Stuff comes as it comes, one shouldn't think about it a lot or intervene consciously.
